src/Orchard.Web/Modules/Orchard.Projections/Providers/SortCriteria/ContentFieldsSortCriteria.cs

```diff
@@ -79,11 +79,11 @@ public class ContentFieldsSortCriterion : ISortCriterionProvider {
 
             // apply where clause
             context.Query = context.Query.Where(relationship, predicate);
-            
+
             // apply sort
-            context.Query = ascending 
-                ? context.Query.OrderBy(relationship, x => x.Asc("Value")) 
-                : context.Query.OrderBy(relationship, x => x.Desc("Value"));
+            context.Query = ascending
+                ? context.Query.OrderBy(relationship, x => x.Asc(context.VersionScope == QueryVersionScopeOptions.Latest ? "LatestValue" : "Value"))
```

Comment:
Maybe create a method in the context to get the field name (based on the scope). I have seen this multiple times.

Reply:
Ok, i will try